performance-tests/commoncrawl-project/collect-stats.sh

```diff
@@ -0,0 +1,612 @@
+#!/bin/bash
```

Comment:
I am thinking that since we have no need for this file for local dev/testing, perhaps it can be added to the project when it's used in the environment for doing real performance testing. Since we don't need it yet for the immediate goal of local dev/testing, let's remove it.